libr/core/cmd_search.c

```diff
@@ -908,10 +908,69 @@ static RList* construct_rop_gadget(RCore *core, ut64 addr, ut8 *buf, int idx,
 	return hitlist;
 }
 
+static int rop_classify_nops (RCore *core, RList *ropList) {
+	char *cmd, *esil_str;
+	int changes = 1;
+	RListIter *iter, *iter_r;
+	RRegItem *item;
+	RList *head;
+	RHashTable *ht_old = r_hashtable_new ();
+	RHashTable *ht_new = r_hashtable_new ();
+
+	srand (time (NULL));
```

Comment:
wtf this is a bad way of seeding the random engine, you have functions in r_util for this, and i dont see a reason to use randomized stuff